site/en/_partials/privacy-sandbox/ps-implementation-status.md

```diff
@@ -0,0 +1,5 @@
+- The [Topics API proposal](https://github.com/jkarlin/topics) has been through a [public discussion](https://github.com/jkarlin/topics/issues) phase  and is now available in an [origin trial](https://github.com/jkarlin/topics/issues). The API is now approaching launch for scaled adoption. 
```

Comment:
```suggestion
- The [Topics API proposal](https://github.com/jkarlin/topics) has completed the public discussion phase and is currently available in an [origin trial](/docs/privacy-sandbox/unified-origin-trial/). The API is now approaching launch for scaled adoption. 
```

Reply:
We don't need to link to the Topics Issues in this bullet point, as the purpose is to talk about the proposal lifecycle. Especially, as we link to feedback below.